changelog.md

```diff
@@ -1,5 +1,6 @@
 ## What's new?
 
+  * Addition of Russian translation
```

Comment:
The changes are listed from oldest to newest. Can you move this line to the bottom?